charts/castai-kvisor/templates/rbac.yaml

```diff
@@ -110,6 +127,48 @@ rules:
       - pods/log
     verbs:
       - get
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - list
```

Comment:
Why do we need list and watch to all secrets in kvisord namespace? Can it be scoped to certsSecret only?

Reply:
its just how cert rotator work, it lists and watches all secrets in namespace